test/CMakeLists.txt

```diff
@@ -69,6 +69,8 @@ nextcloud_add_test(SelectiveSync "")
 nextcloud_add_test(DatabaseError "")
 nextcloud_add_test(LockedFiles "../src/gui/lockwatcher.cpp")
 nextcloud_add_test(FolderWatcher "${FolderWatcher_SRC}")
+nextcloud_add_test(Capabilities "")
+nextcloud_add_test(PushNotifications "mockwebsocketserver.cpp")
```

Comment:
I'd lean toward stuffing the fake websocket server into the syncenginetestutils library (although it wouldn't quite match the name in some way).